mslice/presenters/cut_presenter.py

```diff
@@ -238,6 +240,11 @@ def _cut_axis_changed(self):
                 self._cut_view.populate_input_fields(saved_parameters)
         min_step = self._minimumStep[self._cut_view.get_cut_axis()]
         self._cut_view.set_minimum_step(min_step)
+        self._cut_view.update_integration_axis()
+
+
+    def _integration_axis_changed(self):
```

Comment:
This seems to be an empty method. Is there some missing functionality?

Reply:
Ah, no. It was a place-holder I forgot to delete - I was thinking of putting a callback to check the validity of the new integration axis, but then decided that the code should be in the callback for the cut axis (it repopulates the list of integration axes with allowed values).